### PR TITLE
(libreoffice-streams) Add NoDesktopIcon package parameter

### PR DESCRIPTION
**Pull request by slycordinator**
## Description
Add '/NoDesktopIcon' package parameter to package
Setting --params '/NoDesktopIcon' appends 'CREATEDESKTOPLINK=0' to the end of the package's silentArgs variable.
When the parameter is unset, nothing happens because CREATEDESKTOPLINK=1 is set by default in libreoffice MSI installers.

## Motivation and Context
Allows user to disable the creation of the Desktop Icon / Desktop Shortcut.

## How Has this Been Tested?
I installed and uninstalled libreoffice-fresh and libreoffice-still packages with the --params '/NoDesktopIcon' set and unset, both in a running system and in the chocolatey-community test environment.

With '/NoDesktopIcon' unset, the packages installed with a desktop icon added, as expected.
With '/NoDesktopIcon' set, they installed with no desktop icon.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Migrated package (a package has been migrated from another repository)

## Checklist:

- [X] My code follows the code style of this repository.
- [X] My change requires a change to documentation (this usually means the notes in the description of a package).
- [X] I have updated the documentation accordingly (this usually means the notes in the description of a package).
- [X] I have updated the package description and it is less than 4000 characters.
- [X] All files are up to date with the latest [Contributing Guidelines](https://github.com/chocolatey-community/chocolatey-packages/blob/master/CONTRIBUTING.md)
- [X] The added/modified package passed install/uninstall in the chocolatey test environment.
- [ ] The changes only affect a single package (not including meta package).

### CODE REVIEW

**Comment by pauby** [comment]
@slycordinator Can you point to the documentation that lists this as an acceptable silent installer argument?

**Comment by slycordinator** [comment]
https://wiki.documentfoundation.org/Deployment_and_Migration

CREATEDESKTOPLINK (default=1) - create desktop shortcut icon (bugged in
4.1.0+, solved in tdf#65102 for 4.1.4 and 4.2.0)

File: automatic/libreoffice-streams/libreoffice-streams.nuspec
Comment (pauby) [inline automatic/libreoffice-streams/libreoffice-streams.nuspec:1]: This file doesn't need to be amended as it is updated automatically.

**Review by pauby — CHANGES_REQUESTED** [review]
Unfortunately, I can't test this, as locally I get this error:

```
PS C:\Users\WDAGUtilityAccount\Desktop\chocolatey-coreteampackages\automatic\libreoffice-streams> .\update.ps1
libreoffice-streams - checking updates using au version 2022.10.24

*** Stream: 7.5.3.fresh ***
WARNING: Invalid nuspec file Version '' - using 0.0
URL check
  https://downloadarchive.documentfoundation.org/libreoffice/old/7.5.3.2/win/x86/LibreOffice_7.5.3.2_Win_x86.msi
URL syntax is invalid:
At C:\Program Files\WindowsPowerShell\Modules\au\Public\Update-Package.ps1:110 char:67
+ ... heck_url $url -Options $Latest.Options) { throw "${res}:$url" } else  ...
+                                               ~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (URL syntax is invalid::String) [], RuntimeException
    + FullyQualifiedErrorId : URL syntax is invalid:
```

**Pull request closed without merging**